### ISSUE TITLE
Provide DataLoader registry

### ISSUE THREAD

**Issue by bkuberek** [networknt/light-graphql-4j#66]
Currently, the `GraphqlPostHandler` constructs an `ExecutionInput` and executes the graphql query without providing an option for user provided `DataLoaderRegistry` 
https://github.com/networknt/light-graphql-4j/blob/808682f8ab425c6186937cb1fdc333421cf1013d/graphql-router/src/main/java/com/networknt/graphql/router/handlers/GraphqlPostHandler.java#L89

When using [`DataLoader`](https://www.graphql-java.com/documentation/v12/batching/) we need to set `dataLoaderRegistry`

```
ExecutionInput executionInput = newExecutionInput()
    .query(query)
    .operationName(operationName)
    .context(exchange)
    .root(exchange)
    .variables(variables)
    .dataLoaderRegistry(registry) // <---
    .build()
```
----

Alternatively, we should be able to use `Instrumentation.instrumentExecutionInput(input, parameters)`, but there is a bug in `graphql` https://github.com/graphql-java/graphql-java/issues/1667

It is still possible to do it via instrumentation but I think data loader registry could be added to router.

**Comment by stevehu** [comment]
I totally agree. This is a new feature added to the graphql-java lately. Would you be able to submit a PR for this? Thanks.
